File: lib/tmdb_api/model/production_company.dart
class ProductionCompany {
  int id;
  String name;
  String logoPath;
  String originCountry;

  ProductionCompany({
    this.id,
    this.name,
    this.logoPath,
    this.originCountry,
  });

  @override
  String toString() {
    return name;
  }

  factory ProductionCompany.fromJson(Map<String, dynamic> json) {
    return ProductionCompany(
        id: json['id'],
        name: json['name'],
        logoPath: json['logo_path'],
        originCountry: json['origin_country']);
  }
}

File: kmp/model/src/main/java/com/tikal/tmdb/model/ProductionCompany.kt
package com.tikal.tmdb.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class ProductionCompany(
    @SerialName("id")
    val id: Long,
    @SerialName("name")
    val name: String,
    @SerialName("logo_path")
    val logoPath: String? = null,
    @SerialName("origin_country")
    val originCountry: String? = null
)